From ca2edcf8fd394f77005ed5a4af4aef3b93fc5a1c Mon Sep 17 00:00:00 2001
From: Xavier Boudet <x-boudet@ti.com>
Date: Fri, 14 Sep 2012 15:53:08 +0200
Subject: [PATCH 2/2] Update to libudev.pc name

Signed-off-by: Xavier Boudet <x-boudet@ti.com>
---
 Makefile.am  |    4 ++--
 configure.ac |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

Index: libgbm/Makefile.am
===================================================================
--- libgbm.orig/Makefile.am	2012-09-14 16:32:27.203312896 +0200
+++ libgbm/Makefile.am	2012-09-14 16:32:30.691312979 +0200
@@ -6,10 +6,10 @@
 	common.c
 
 libgbm_la_LIBADD =	\
-	@UDEV_LIBS@
+	@LIBUDEV_LIBS@
 
 libgbm_la_CFLAGS =	\
-	@UDEV_CFLAGS@	\
+	@LIBUDEV_CFLAGS@	\
 	-D_OS_UNIX=1	\
 	-DMODULEDIR='"$(exec_prefix)/lib/gbm"'
 
Index: libgbm/configure.ac
===================================================================
--- libgbm.orig/configure.ac	2012-09-14 16:32:27.203312896 +0200
+++ libgbm/configure.ac	2012-09-14 16:35:09.163316758 +0200
@@ -35,8 +35,8 @@
 AC_PROG_LIBTOOL
 
 # Obtain compiler/linker options for dependencies
-PKG_CHECK_MODULES(UDEV, udev)
-PKG_CHECK_MODULES(DRM, libdrm)
+PKG_CHECK_MODULES([LIBUDEV], [libudev])
+PKG_CHECK_MODULES([DRM], [libdrm])
 
 AC_CONFIG_FILES([Makefile gbm.pc])
 AC_OUTPUT
